src/PointwiseFunctions/GeneralRelativity/ComputeGhQuantities.hpp

```diff
@@ -1113,7 +1113,8 @@ template <size_t SpatialDim, typename Frame>
 struct ThreeIndexConstraintCompute : ThreeIndexConstraint<SpatialDim, Frame>,
                                      db::ComputeTag {
   using argument_tags =
-      tmpl::list<gr::Tags::DerivSpacetimeMetric<SpatialDim, Frame>,
```

Comment:
Why does this tag exist?  Shouldn't it be removed?

Reply:
It's not used for anything as far as I can see, so I have no objection to removing it

Reply:
yes, please remove it as I don't know what it is